docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -429,6 +429,19 @@ dayjs.extend(customParseFormatPlugin);
 
 The other plugins are still added before the adapter initialization.
 
+## Remove root level `locales` export
+
+The `locales` export has been removed from the root of the packages.
+In an effort to reduce the bundle size, the locales are now only available from the `@mui/x-date-pickers/locales` or `@mui/x-date-pickers-pro/locales` paths.
+If you were still relying on the root level export, please update your code.
+
+Before v7, it was possible to import locales from the package root (i.e. `import { frFR } from '@mui/x-date-pickers'`).
+
+```diff
+- import { frFR } from '@mui/x-date-pickers';
++ import { frFR } from '@mui/x-date-pickers/locales';
```

Comment:
Same as https://github.com/mui/mui-x/pull/11650#discussion_r1448084598

```suggestion
-import { frFR } from '@mui/x-date-pickers';
+import { frFR } from '@mui/x-date-pickers/locales';
```

Reply:
Thanks for bringing it up.
Raised a PR fixing it and some other tiny issues. 😉 
https://github.com/mui/mui-x/pull/11695